Blackjack/CardRank.cs

```diff
@@ -24,8 +25,11 @@ public enum CardRank
         Nine,
         [Description("10")]
         Ten,
+        [Description("Jack")]

```

Comment:
You can associate an enum value with a specific int value e.g. `Jack = 11` , then you can try casting it to an int as mentioned in one of your comments to display the number as the output